apps/system/test/unit/app_window_test.js

```diff
@@ -902,6 +902,11 @@ suite('system/AppWindow', function() {
     });
   });
 
+  test('get Private Browser Splash Icon', function() {
+    var app = new AppWindow(fakePrivateConfig);
+    assert.isTrue(app._splash.indexOf('private_browser') !== 1);
```

Comment:
What is this meant to test? My assumption was that it was testing the old private_browser icon path which looks to be updated in this patch. I'm guessing it's catching a different string somewhere, or the URL of the splash page? Maybe add the .png to the filepath?